Session/Conversations/Context Menu/ContextMenuVC+ActionView.swift

```diff
@@ -90,6 +90,14 @@ extension ContextMenuVC {
             themeBackgroundColor = .clear
             
             iconImageView.image = action.icon?.withRenderingMode(.alwaysTemplate)
+            // Flip horizontally for RTL languages
+            iconImageView.transform = CGAffineTransform.identity
+                .scaledBy(
+                    x: (action.flipIconForRTL ? -1 : 1),
+                    y: 1
+                )
+            iconImageView.transform = CGAffineTransform(
+                scaleX: -1.0, y: 1.0)
```

Comment:
Seems like this might have been some debug code mistakenly left in? (it'd override the previous `iconImageView.transform` which is based on the `action.flipIconForRTL`)